README.md

```diff
@@ -8,6 +8,8 @@
 
 Welcome to Braintree's iOS SDK. This library will help you accept card and alternative payments in your iOS app.
 
+📣 **Upgrade your integration to continue accepting Braintree payments** 📣 The SSL certificates for current iOS SDK versions (v5 and v6) are set to expire by June 31, 2025. Upgrade to v5.26.0+ and v6.17.0+, respectively, to continue using the Braintree SDK. ![Click here for more details](https://github.com/braintree/braintree_ios/issues/1277)
```

Comment:
Did we want to move the issue into the discussions section or leave as an issue?

Reply:
_Discussed during standup to pin the issue_